whois-commons/src/main/java/net/ripe/db/whois/common/hazelcast/WhoisHazelcastBlockedIps.java

```diff
@@ -0,0 +1,61 @@
+package net.ripe.db.whois.common.hazelcast;
+
+import com.google.common.base.Joiner;
+import com.hazelcast.collection.ISet;
+import com.hazelcast.core.HazelcastInstance;
+import net.ripe.db.whois.common.ip.IpInterval;
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+@Component
+@Profile({WhoisProfile.DEPLOYED})
+public class WhoisHazelcastBlockedIps implements HazelcastBlockedIps{
+
+    private static final Logger LOGGER = getLogger(WhoisHazelcastBlockedIps.class);
+
+    private static final Joiner COMMA_JOINER = Joiner.on(',');
+
+    private final ISet<IpInterval> ipBlockedSet;
+
+    public WhoisHazelcastBlockedIps(final HazelcastInstance hazelcastInstance,
+                                    @Value("${ipranges.blocked.list:}") final String blockedListIps) {
+
+        ipBlockedSet = hazelcastInstance.getSet("ipBlockedSet");
+        ipBlockedSet.addAll(getBlockedIntervals(blockedListIps));
+
+        LOGGER.info("hazelcast instances {} members: {} " , hazelcastInstance.getName() , hazelcastInstance.getCluster().getMembers());
```

Comment:
remove this logging?

Reply:
It is there because it is there in the `HazelcastPersonalObjectAccounting`class as well.
I will remove it then